web/src/app/administration/components/import-export/import-export.component.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { StartupService } from '../../../shared/services/startup/startup.service';
+import { TaskanaEngineService } from '../../../shared/services/taskana-engine/taskana-engine.service';
+import { WindowRefService } from '../../../shared/services/window/window.service';
+import { DomainService } from '../../../shared/services/domain/domain.service';
+import { WorkbasketDefinitionService } from '../../services/workbasket-definition.service';
+import { NotificationService } from '../../../shared/services/notifications/notification.service';
+import { UploadService } from '../../../shared/services/upload/upload.service';
+import { ImportExportService } from '../../services/import-export.service';
+import { HttpClient, HttpHandler } from '@angular/common/http';
+import { Router } from '@angular/router';
+import { of } from 'rxjs';
+import { ClassificationDefinitionService } from '../../services/classification-definition.service';
+import { take, timeout } from 'rxjs/operators';
+import { TaskanaType } from '../../../shared/models/taskana-type';
+import { BlobGenerator } from '../../../shared/util/blob-generator';
+jest.mock('../../../shared/util/blob-generator');
+
+describe('ImportExportComponent', () => {
+  let fixture: ComponentFixture<ImportExportComponent>;
+  let debugElement: DebugElement;
+  let app: ImportExportComponent;
+
+  const domainServiceSpy = jest.fn().mockImplementation(
+    (): Partial<DomainService> => ({
+      getSelectedDomainValue: jest.fn().mockReturnValue(of()),
+      getSelectedDomain: jest.fn().mockReturnValue(of()),
+      getDomains: jest.fn().mockReturnValue(of())
+    })
+  );
+
+  const httpSpy = jest.fn().mockImplementation(
+    (): Partial<HttpClient> => ({
+      get: jest.fn().mockReturnValue(of([])),
+      post: jest.fn().mockReturnValue(of([]))
+    })
+  );
+
+  const showDialogFn = jest.fn().mockReturnValue(true);
+  const notificationServiceSpy = jest.fn().mockImplementation(
+    (): Partial<NotificationService> => ({
+      showDialog: showDialogFn,
+      showToast: showDialogFn,
+      triggerError: showDialogFn
+    })
+  );
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [],
+      declarations: [ImportExportComponent],
+      providers: [
+        StartupService,
+        TaskanaEngineService,
+        WindowRefService,
+        WorkbasketDefinitionService,
+        ClassificationDefinitionService,
+        UploadService,
+        ImportExportService,
+        { provide: DomainService, useClass: domainServiceSpy },
+        { provide: NotificationService, useClass: notificationServiceSpy },
+        { provide: HttpClient, useClass: httpSpy }
+      ]
+    }).compileComponents();
+
+    jest.clearAllMocks();
+
+    fixture = TestBed.createComponent(ImportExportComponent);
+    debugElement = fixture.debugElement;
+    app = fixture.debugElement.componentInstance;
+    app.currentSelection = TaskanaType.WORKBASKETS;
+    fixture.detectChanges();
+  }));
+
```

Comment:
All our test files start with the simple 'should create component'-test. Maybe you could add this test here, too.